src/App.jsx

```diff
@@ -1,9 +1,12 @@
 import './App.css';
+import SignIn from './pages/SignInPage';
+import SignUp from './pages/SignUpPage';
 
 function App() {
   return (
     <div>
-      <h1 className="text-3xl font-bold underline">Hello world!</h1>
+      <SignIn />
+      <SignUp />
```

Comment:
Can we put those pages inside their own folders also? src/pages/SignInPage/<your-filename>